src/main/java/org/jitterbit/connector/dropbox/activities/ProcessFileActivity.java

```diff
@@ -154,6 +164,61 @@ public void execute(JitterbitActivity.ExecutionContext context) throws ActivityE
     }
   }
 
+
+  protected ActivityRequestResponseMetaData multipleSchemas2() {
```

Comment:
Should this method come second (after `multipleSchemas()` line 200) so that it is in the same order as the creation of the models (lines 284 et al)?

Reply:
Doesn't really matter in this case

Reply:
It's a matter of style and presentation rather than correctness of the code. Just so when you read that method, it lines up in the same way…